.github/scripts/set_benchmark_param.sh

```diff
@@ -33,3 +33,8 @@ if [[ $1 == *"internlm2-chat-20b"* ]] || [[ $1 == *"Qwen1.5-32B-Chat"* ]]
 then
   echo "TP_INFO=--tp 2" >> "$GITHUB_ENV"
 fi
+
+if [[ $1 == *"internlm2"* ]] || [[ $1 == *"Llama-3"* ]]
+then
```

Comment:
Do you mean internlm2 and llama3 share the same context window size?

Reply:
internlm2 and llama3  will do longtext benchmark